docs/siem/detections/api/lists/api-find-list-items.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[lists-api-find-list-items]]
+=== Find list items
+
+Retrieves a paginated subset of list items in the specified container. By
+default, the first page is returned with 20 results per page.
+
+==== Request URL
+
+`GET <kibana host>:<port>/api/lists/items/_find`
+
+===== URL query parameters
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`list_id` |String |ID of the <<lists-api-create-container, list container>>. |Yes
+
+|`page` |Integer |The page number to return. |No
+
+|`per_page` |Integer |The number of items to return per page. |No
+
+|`sort_field` |String |Determines which field is used to sort the results. |No
+
+|`sort_order` |String |Determines the sort order, which can be `desc` or `asc`.
+|No
+|`cursor` |String |Returns the items that come after the last item
+returned in the previous call (use the `cursor` value returned in the previous
+call). This parameter uses the `tie_breaker_id` field to ensure all items
+are sorted and returned correctly.
+See {ref}/search-request-body.html#request-body-search-search-after[Search After] for more information. |No
+
+|==============================================
+
+===== Example request
+
+Retrieves list items in the `external-ip-excludes` list container:
+
+[source,console]
+--------------------------------------------------
+GET api/lists/items/_find?list_id=external-ip-excludes
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+    Indicates a successful call.
+    
+==== Response payload
+
+[source,json]
+--------------------------------------------------
+{
+  "cursor": "WzIwLFsiOWI2YWViMDctZjkzYy00YjE3LTk4MWEtZGM1YjI1NjUxZTY0Il1d",
+  "data": [
+    {
+      "created_at": "2020-07-09T08:43:22.318Z",
+      "created_by": "LiverpoolFC",
+      "id": "melwood-static-ip",
+      "list_id": "external-ip-excludes",
+      "tie_breaker_id": "25ce0bef-8b9e-42fe-a958-bbe7fced4588",
+      "type": "ip",
+      "updated_at": "2020-07-09T08:43:22.318Z",
+      "updated_by": "LiverpoolFC",
+      "value": "107.96.164.225"
+    },
+    {
+      "created_at": "2020-07-09T08:44:31.155Z",
+      "created_by": "LiverpoolFC",
+      "id": "anfield-static-ip",
+      "list_id": "external-ip-excludes",
+      "tie_breaker_id": "9b6aeb07-f93c-4b17-981a-dc5b25651e64",
+      "type": "ip",
+      "updated_at": "2020-07-09T08:44:31.155Z",
+      "updated_by": "LiverpoolFC",
+      "value": "37.138.5.225"
+    }
+  ],
+  "page": 1,
+  "per_page": 20,
+  "total": 2
+}
```

Comment:
This has just the additional field of `_version` for items for OCC

```ts
❯ ./find_list_items.sh
{
  "cursor": "WzIwLFsiY2U3OGQ5ZDAtM2RmZS00OWRiLTg3MTgtMzFmZTMxODJjZDcxIl1d",
  "data": [
    {
      "_version": "WzUyMjMwMTUsMV0=",
      "created_at": "2020-08-04T21:04:11.940Z",
      "created_by": "yo",
      "id": "ip_item",
      "list_id": "ip_list",
      "tie_breaker_id": "ce78d9d0-3dfe-49db-8718-31fe3182cd71",
      "type": "ip",
      "updated_at": "2020-08-04T21:04:11.940Z",
      "updated_by": "yo",
      "value": "127.0.0.1"
    }
  ],
  "page": 1,
  "per_page": 20,
  "total": 1
}
```

and it makes sense as we keep the `immutable` and `version` at the top level list level but do not update the list items with that information. Also, when we update a list item we do not update the version number at the list item level but only if the list top level changes at the moment.